packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -146,9 +146,9 @@ contract OPContractsManager is ISemver {
 
     // -------- Constants and Variables --------
 
-    /// @custom:semver 1.6.0
+    /// @custom:semver 1.6.1
     function version() public pure virtual returns (string memory) {
-        return "1.6.0";
+        return "1.6.1";
```

Comment:
Do we need a semver bump here, what code changed in OPCM?

Reply:
I'm not sure what changed but the semver-lock entry changed